app/routers/weather_forecast.py

```diff
@@ -0,0 +1,272 @@
+import datetime
```

Comment:
That's should be in `app/internals/weather.py` (sorry if I said other thing in the past).

We don't want to give the users the option to use our API key for weather requests.

Reply:
I'm on it. Few min and I'll push it again. 10x.